tests/data/Q748282-en/output.html

```diff
@@ -1,93 +1,18 @@
-  </li>
-  <li>
-    Kemal-Egerek (Ukrainian: <span lang="uk">Кемаль-Егерек</span>, Russian: <span lang="ru">Кемаль-Эгерек</span>, Crimean Tatar: <span lang="crh">Kemal Egerek</span>) 1,529 m in the Babugan Yayla;
-  </li>
-  <li>
-    Eklizi-Burun (Ukrainian: <span lang="uk">Еклізі-Бурун</span>, Russian: <span lang="ru">Эклизи-Бурун</span>, Crimean Tatar: <span lang="crh">Eklizi Burun</span>) 1,527 m in the Chatyrdag Yayla;
-  </li>
-  <li>
-    Lapata (Ukrainian: <span lang="uk">Лапата</span>; Russian: <span lang="ru">Лапата</span>, Crimean Tatar: <span lang="crh">Lapata</span>) 1,406 m in the Yaltynska Yayla, Yalta Yaylası;
-  </li>
-  <li>
-    Northern Demirji (Ukrainian: <span lang="uk">Північний Демірджі</span>, Russian: <span lang="ru">Северный Демирджи</span>, Crimean Tatar: <span lang="crh">Şimaliy Demirci</span>) 1,356 m in the Demirci Yayla;
-  </li>
-  <li>
-    Ai-Petri (Ukrainian: <span lang="uk">Ай-Петрі</span>, Russian: <span lang="ru">Ай-Петри</span>, Crimean Tatar: <span lang="crh">Ay Petri</span>) 1,234 m in the Ay Petri Yaylası.
-  </li>
-</ul>
-<h2>
-  Passes and rivers
-</h2>
-<p>
-  The passes over the Crimean Mountains are: (from east to west)
-</p>
-<ul>
-  <li>
-    Angarskyi Pass (752m) near Perevalne, on a road from Alushta to Simferopol
-  </li>
-  <li>
-    Baydar Gate (503m) near Foros, connecting Baydar Valley and the sea coast
-  </li>
-  <li>
-    Laspi Pass (350m) near Cape Aya, on a road from Yalta to Sevastopol.
-  </li>
-</ul>
-<p>
-  Rivers of the Crimean Mountains include the Alma River, Chernaya River, and Salhir River on the northern slope and Uchan-su River on the southern slope which forms the Uchan-su waterfall, and the highest waterfall in Crimea.
-</p>
-<h2>
-  History
-</h2>
-<p>
-  Archaeologists have found the earliest anatomically modern humans in Europe in the Crimean Mountains' Buran-Kaya caves. The fossils are 32,000 years old, with the artifacts linked to the Gravettian culture. The fossils have cut marks suggesting a post-mortem defleshing ritual.
-</p>
\ No newline at end of file
+<p>The <b>Crimean Mountains</b> (Crimean Tatar: <span lang="crh">Qırım dağları</span>; Ukrainian: <span lang="uk">Кримські гори</span>; Russian: <span lang="ru">Крымские горы</span>; Turkish: <i lang="tr">Yayla Dağları</i>) or <b>Yayla Mountains</b> are a range of mountains running parallel to the south-eastern coast of Crimea, between about 8–13 kilometers (5–8 miles) from the sea. Toward the west, the mountains drop steeply to the Black Sea, and to the east, they change slowly into a steppe landscape.</p><p>The Crimean Mountains consist of three subranges. The highest is the Main Range, which is subdivided into several yaylas or mountain plateaus (<i>yayla</i> or <i>yaylak</i> is Turkic for "alpine meadow"). They are:</p><ul><li>Baydar yayla</li>
+<li>Ai-Petri yayla</li>
```

Comment:
Is there any minification used later, to remove unnecessary line endings for final HTML pages?

Reply:
See above - we don't do a proper minification step, so whitespace _within_ elements is left in the output.